cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -23,9 +23,11 @@ OPTIONS
 
        --mode client | service | proxy
```

Comment:
```suggestion
       --mode client | service
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/fdbf3d28db757767a7ef880cc166507ea51caaa7